crates/rspack_plugin_javascript/tests/fixtures/system/test.config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "output": {
+    "library": {
```

Comment:
Please not add some library test at rust side, it is ok because you already port webpack js test.

Reply:
done